NEWS.md

```diff
@@ -38,14 +38,21 @@ Standard library changes
   * `edit` can now be called on a module to edit the file that defines it ([#29636]).
   * `diff` now supports arrays of arbitrary dimensionality and can operate over any dimension ([#29827]).
   * `sprandn` now supports result types like `ComplexF64` or `Float32` ([#30083]).
+  * `parse` now allows strings representing integer 0 and 1 for type `Bool` ([#29980]).
+  * `tail` now works on named tuples ([#29595]).
```

Comment:
`tail` (IMO unfortunately) isn't exported, so this should probably be mentioned as `Base.tail`.